### DISCUSSION TITLE
Alternative more powerful hardware than a Raspberry Pi

### DISCUSSION THREAD

**Discussion by SepticFrank** [raspiblitz/raspiblitz#4635] [General]
I'm interested in installing raspiblitz on more powerful hardware, around a $500-$750 budget. But I'm also not a super advanced user so easier to configure is better. Any recommendations on a shopping list and guide? 

I was looking at something like this as a starting point but I'm open to suggestions. https://www.amazon.com/dp/B0CWTT32RT?ref=ppx_yo2ov_dt_b_product_details&th=1

**Comment by rootzoll** [comment]
I would recommend looking into an old/used Thinkpad laptop and update it with a 2TB ssd. Making RaspiBlitz easy to use with such old laptops is one goal for RaspiBloitz v1.12.0 that might come later this year - so you can help to teast early and give feedback.
